src/canvas/quadFeature.js

```diff
@@ -143,5 +143,5 @@ var canvas_quadFeature = function (arg) {
 inherit(canvas_quadFeature, quadFeature);
 
 // Now register it
-registerFeature('canvas', 'quad', canvas_quadFeature);
+registerFeature('canvas', 'quad', canvas_quadFeature, {clr: false, img: true, 'img-full': false});
```

Comment:
@manthey how do control and maintain this vocabulary? May we should make it part of public interface of baseclass (keys) and then the derived classes can just refer to it like this: 

registerFeature('canvas', 'quad', canvas_quadFeature, {quadFeature.color: false, quadFeature.img: true, quadFeature.img-full: false});

thoughts?